code/modules/atmospherics/machinery/components/trinary_devices/mixer.dm

```diff
@@ -74,36 +74,36 @@
 
 	var/general_transfer = (target_pressure - output_starting_pressure) * air3.volume / R_IDEAL_GAS_EQUATION
 
-	var/transfer_moles1 = node1_concentration * general_transfer / air1.temperature
+	var/transfer_moles1 = node1_concentration ? node1_concentration * general_transfer / air1.temperature : 0
```

Comment:
It's to prevent the 0/0 case, given air1.temperature can be 0 if node1_concentration is 0. It might make it clearer if I changed it to be air1.temperature?, but it's equivalent.

Reply:
you should add a comment to that effect then

Reply:
I changed it to air1.temperature? , which I think is clear enough to not need commenting.

Reply:
yeah I left this comment before noticing the outdated (hate that github doesn't autohide outdated convos)